core/arch/arm/arm.mk

```diff
@@ -207,6 +207,11 @@ endif
 # this in platform config if different.
 CFG_MAX_CACHE_LINE_SHIFT ?= 6
 
+# CFG_EXTERNAL_ABORT_PLAT_HANDLER is used to implement platform-specific
+# handling of external abort implementing the plat_external_abort_handler()
+# function.
+CFG_EXTERNAL_ABORT_PLAT_HANDLER ?= n
```

Comment:
The should probably go into `mk/config.mk`.

Reply:
Sure, I didn't think of other architecture since I've only modified ARM abort.c.
I'll update the P-R